data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,210 @@ mission "Arachi Cartoonist 2"
+			label representatives
+			scene "scene/councilofahr1"
+			`	He shows you a data pad, pointing to three symbols on the screen as he names the houses from left to right. "Issues with her other transport, the representative of House Bebliss had, so waiting on Stronghold of Flugbu, she still is. Word that House Chamgar is ready for the Council, I just received, and on Hammer of Debrugt, their member is. Spare some time for the Council, House Ablomab finally can, and waiting for you on Delve of Bloptab, their envoy is.`
+			`	"Receive your payment, you will, once on <planet>, the three representatives are. I shall inform the houses that expect you, they should. Thankful for your assistance, the royal family is."`
+			`	He briefly bows his head while stretching his legs in some sort of salute, then disappears into the spaceport. Once you're at your ship, you place markers on <stopovers>, where the representatives are waiting.`
+				accept
+	to complete
+		has "Coalition: Council of Ahr: Ablomab: done"
+		has "Coalition: Council of Ahr: Bebliss: done"
+		has "Coalition: Council of Ahr: Chamgar: done"
+	on abort
+		fail "Coalition: Council of Ahr: Ablomab"
+		fail "Coalition: Council of Ahr: Bebliss"
+		fail "Coalition: Council of Ahr: Chamgar"
+		fail "Coalition: Council of Ahr: Drama"
+	on complete
+		"coalition jobs" += 5
+		payment 4500000
+
+mission "Coalition: Council of Ahr: Ablomab"
+	invisible
+	landing
+	passengers 1
+	blocked "You have reached <origin>, but you need <capacity> in order to take on the next mission. Return here when you have the required space free."
+	to offer
+		has "Coalition: Council of Ahr: active"
+	source "Delve of Bloptab"
+	destination "Ahr"
+	on offer
+		conversation
+			`You land on <source> to find that House Ablomab's representative had to oversee an "emergency meltdown drill" on the neighboring planet, Bloptab's Furnace. His interplanetary shuttle still hasn't arrived, so you're left to wait for him on the spaceport.`
+			`	He finally arrives some hours after sunset, dragging along a heavy suitcase and barely holding on to what must be the Arach equivalent of coffee. As you help him with the heavy luggage, he introduces himself as Dubug. "Sorry about the wait, I am. Replace a coworker on the meltdown drill, I had to. A routine task, it is, but also routine, the Council is, so care much for the delay, the house did not. Show me to my room, could you?"`
+			`	While you're both walking to your ship, he finishes his drink, but it doesn't seem to help him keep his eyes open; you have to help him avoid bumping into other Arachi here, or the spaceport decorations. Although he doesn't seem very talkative, it might help his focus to get a conversation going.`
```

Comment:
```suggestion
			`	While you're both walking to your ship, he finishes his drink, but it doesn't seem to help him keep his eyes open; you have to help him avoid bumping into either other Arachi here or the spaceport decorations. Although he doesn't seem very talkative, it might help his focus to get a conversation going.`
```